ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/representative-journey.spec.ts

```diff
@@ -113,27 +114,25 @@ describe('RepresentativeJourney', () => {
   };
 
   it(`should continue to ${Steps.QuestionnaireCompleted} if representative has no access to a computer`, () => {
-    givenUserIsAtStep(Steps.AccessToComputer);
     journey.model.computer = false;
-    journey.next();
     expectDropOffToQuestionnaireCompletedFrom(Steps.AccessToComputer);
   });
+
   it(`should continue to ${Steps.ContactUs} from ${Steps.QuestionnaireCompleted} if representative has no access to a computer`, () => {
-    givenUserIsAtStep(Steps.QuestionnaireCompleted);
     journey.model.computer = false;
-    journey.next();
     expectDropOffToContactUsFrom(Steps.QuestionnaireCompleted);
   });
-  it(`should continue to ${Steps.QuestionnaireCompleted} if representative has no camera`, () => {
-    givenUserIsAtStep(Steps.AboutYourComputer);
-    journey.model.camera = HasAccessToCamera.No;
-    journey.next();
-    expectDropOffToQuestionnaireCompletedFrom(Steps.AboutYourComputer);
+
+  it(`should continue to ${Steps.QuestionnaireCompleted} if answered representative has answered question about camera`, () => {
+    for (const answer of [ HasAccessToCamera.No, HasAccessToCamera.Yes, HasAccessToCamera.No ]) {
+      journey.forSuitabilityAnswers(suitabilityAnswers.oneUpcomingHearing);
+      journey.model.camera = answer;
+      expectDropOffToQuestionnaireCompletedFrom(Steps.AboutYourComputer);
+    }
   });
+
   it(`should continue to ${Steps.ContactUs} from ${Steps.QuestionnaireCompleted} if representative has no camera`, () => {
-    givenUserIsAtStep(Steps.QuestionnaireCompleted);
```

Comment:
the `expectDropOffToContactUsFrom` method is already setting the current step as well as calling `journey.next` so I don't think it shouldn't be done twice